app/utils/filters.js

```diff
@@ -47,6 +47,9 @@ function describeFilters (filters) {
 
   const brandStates = Array.isArray(filters.brand) ? filters.brand.map(brand => brand.replace('-', ' ')) : []
   if (brandStates.length === 0 && filters.brand) {
+    if (filters.brand === 'jcb') {
+      filters.brand = 'JCB'
+    }
```

Comment:
Maybe we could fix this so it's coming from the downstream service in the right format, and then we can go back and fix this so we don't need to do that.